.github/workflows/ci.yml

```
@@ -13,50 +13,53 @@
+        run: |
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics || exit 1
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88 --statistics
+
+      # Run Gradio interface test
+      - name: Test Gradio app
+        run: |
+          python -m unittest discover -s tests
+
+      # (Optional) Cache dependencies
+      - name: Cache dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)